src/Content/Changes.php

```diff
@@ -23,86 +27,112 @@
 class Changes
 {
 	/**
-	 * Access helper for the field, in which changes are stored
+	 * Access helper for the cache, in which changes are stored
 	 */
-	public function field(): Field
+	public function cache(): Cache
 	{
-		return $this
-			->site()
-			->version(VersionId::published())
-			->content()
-			->get('changes');
+		return App::instance()->cache('changes');
```

Comment:
`Suggestion:` Since we keep using `App::instance()` around the class, why not define this once in the constructor?

```suggestion
		return $this->kirby->cache('changes');
```